conduct.md

```diff
@@ -134,14 +134,14 @@ TODO: Write an introductory paragraph.
 
 TODO: Write an introductory paragraph.
 
-#### Acceptable Conduct
+#### Level 1 Infraction
 
-- TODO
-- TODO
-- TODO
+- Students who commit violation once the will be recorded by professor and warned orally.
 
-#### Unacceptable Conduct
+#### Level 2 Infraction
 
-- TODO
-- TODO
-- TODO
+- Students who commit violation 3 times or more in total will be required to have a meeting with professor or TLs to talk about their infractions.
+
+#### Level 3 Infraction
+
+- Students who commit violation 5 times in total or avoid to meet with professor will be punished by deducting grades on their current assignment.
```

Comment:
Should these be written in paragraph form, rather than bullet form?

Reply:
"avoid to meet" should be "avoid meeting". Also, it should be specified that a level 3 infraction will occur if a student avoids meeting with the professor after a level 2 infraction.

Reply:
Instead of "be punished", could it just say, "have points deducted from their grade on the current assignment"? "Punished" is quite a strong word.